src/Pages/yongseok/Main/Nav/Nav.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
+import ProfileModal from '../Components/ProfileModal/ProfileModal';
+import SearchModal from '../Components/SearchModal/SearchModal';
+import './Nav.scss';
+
+class Nav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      dummyData: [],
+      filterData: [],
+      searchModalShowing: false,
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/data.json', {
+      method: 'GET',
+    })
```

Comment:
get메소드는 default 값으로 생략하여 아래와 같이 사용할 수 있습니다.
```suggestion
    fetch('http://localhost:3000/data/data.json')
```

Reply:
넵 !! 삭제완료 했습니다 !